integration-tests/tests/create-application-from-git-source.spec.ts

```diff
@@ -0,0 +1,44 @@
+// <reference types="cypress" />
+import { AddComponentPage } from '../support/pages/AddComponentPage';
+import { ApplicationComponentPage } from '../support/pages/ApplicationComponentsPage';
+import { ComponentSamplesPage } from '../support/pages/ComponentSamplesPage';
+import { CreateApplicationPage } from '../support/pages/CreateApplicationPage';
+import { BuildApplicationWithGitSourcePage } from '../support/pages/BuildApplicationWithGitSourcePage';
+import { Common } from '../utils/Common';
+
+describe('Create Application from Git Source', () => {
```

Comment:
it's the component that is being created from git, not the application, so the terminology is a bit off

Reply:
Makes sense :+1: thanks